docs/dev_guide/aerospike_setup_guide.md

```diff
@@ -64,4 +64,36 @@ Next we will verify the functionality of Aerospike database by performing basic
 	
 	# View the config
 	asadm -e "show config"
-```
\ No newline at end of file
+```
+
+# Configure feathr core Spark to connect with Aerospike
+
+1. To connect to Aerospike (with Aerospike SDK, or Spark), username and password need to be configured. 
+Guidance for setting up username and password:
+https://docs.aerospike.com/server/operations/configure/security/access-control
+
+2. To connect to Aerospike with Spark, a spark conector jar needs to be submitted to your Spark runtime. 
+Link to spark connector: 
+https://docs.aerospike.com/connect/spark
+
+3. To operate data in Aerospike database via Spark, spark job should have following configurations:
```

Comment:
I think this doc is about how to configuring AS, not just for feathr.  To write to AS, any spark job need to have the configs.

Reply:
The doc says `spark job should have following configurations`, which is obviously not about how to config AS.
These configurations are passed from the Feathr client to Feathr Job on Spark, these config/options are hidden to the end users.
The doc should tell user how to create `AeroSpikeSink` instead of these hidden settings, because the user has no way to config them.
A code snippet about `AeroSpikeSink` creation will be helpful.

Reply:
Added~